great-idea-website/index.html

```diff
@@ -1,63 +1,94 @@
 <!doctype html>
 
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
 
   <title>Great Idea!</title>
 
-  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Bangers%7CTitillium+Web" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
 
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
 
-  <body>
+<body>
+  <nav>
+    <a href="">Services</a>
+    <a href="">Product</a>
+    <a href="">Vision</a>
+    <a href="">Features</a>
+    <a href="">About</a>
+    <a href="">Contact</a>
+    <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+  </nav>
 
-  Services
-  Product
-  Vision
-  Features
-  About
-  Contact
 
-  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-  Innovation
-  On
-  Demand
 
-  Get Started
 
-  <img src="img/header-img.png" alt="Image of a code snippet.">
 
-  Features
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+  <header>
+    <div>
+      <h1 class= "Headerone">
+        Innovation
+        On
+        Demand
+      
+        Get Started
+      </h1>
+      <button class= "Button">
+          <img src="img/header-img.png" alt="Image of a code snippet.">
 
+      </button>
```

Comment:
**You need the above image tag to NOT be in the button tag.** 

Read this article on button tags:
[MDN HTML button tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button)

Really, seriously, lookup, read and understand all of these tags if you do not already understand them! _You will need to be able to understand what every element here does in order to move forward._ 

If you don't know EXACTLY how something like button works, goto google, search "MDN HTML button (or whatever it is you don't get)".